content/preferences.html

```diff
@@ -225,7 +224,6 @@
 	<script type="application/x-javascript" src="./shortcuts-and-tweaks/transfer-search-filters.js"></script>
 	<script type="application/x-javascript" src="./shortcuts-and-tweaks/transfer-search-result-filters.js"></script>
 	<!-- end categorized modules -->
-
```

Comment:
this file (and others) should not be edited manually, we have `module-update.py` for it